themes-default/slim/views/home.mako

```diff
@@ -14,16 +14,13 @@ window.app = {};
 const startVue = () => {
     window.app = new Vue({
         store,
+        router,
         el: '#vue-wrap',
-        metaInfo: {
-            title: 'Home'
-        },
-        data() {
-            return {
-                header: 'Show List'
-            };
-        },
-        computed: Object.assign({
+        computed: {
+            isRoute() {
+                const { matched } = this.$route;
+                return matched.filter(route => route.components.default !== undefined).length >= 1;
```

Comment:
What's this?

Reply:
It's for the future. It returns `true` if we have a matched component vs a matched route.

Reply:
For the home "partials"?

Reply:
No for determining if you're in a Vue component or a mako template.